pfcpiface/bess.go

```diff
@@ -647,7 +647,9 @@ func (b *bess) addPDR(ctx context.Context, done chan<- bool, p pdr) {
 				intEnc(p.fseID),         /* fseid */
 				intEnc(uint64(p.ctrID)), /* ctr_id */
 				intEnc(uint64(p.qerID)), /* qer_id */
-				intEnc(uint64(p.farID)), /* far_id */
+				// TODO
+				intEnc(uint64(p.qerID + 1)), /* qer_id2 */
```

Comment:
I suggest setting a default value to skip metering.  I think @badhrinathpa is planning to work on PFCP agent changes to handle `qer_id2` by parsing a list of QERs from the control plane, so this will come in a different PR.

Not sure if 0 is a valid QER ID, this is just an example:
```suggestion
				intEnc(uint64(0)), /* qer_id2 */
```

Reply:
superseeded by using only the fseid